documentation/modules/new-features-and-enhancements-2-7.adoc

```diff
@@ -13,3 +13,17 @@ Possible options are:
 * SCSI
 * SATA
 * VirtIO
+
+// https://issues.redhat.com/browse/MTV-2029
+* In {project-short} 2.7.11, you can now migrate shared disks from VMware vSphere by using the new label `migrateSharedDisks` in `Plan` CRs.
++
+The label can be set to `true` or `false`:
```

Comment:
hmm I would not use "label" maybe "attribute" or "parameter", the label has k8s meaning and don't want to confuse the users?

Reply:
Will do, but I've been using "label" in the CLI documentation. (!)  Sounds like we should talk about this term  for the user guide as a whole, but I'll only change this and the shared disk section for now.